BingBertSquad/turing/nvidia_modeling.py

```diff
@@ -509,14 +509,14 @@ def __init__(self, config, args):
 
             cuda_config = DeepSpeedTransformerConfig(
                 batch_size=ds_config.train_micro_batch_size_per_gpu,
-                max_seq_length=args.max_seq_length,
                 hidden_size=config.hidden_size,
                 intermediate_size=config.intermediate_size,
                 heads=config.num_attention_heads,
                 attn_dropout_ratio=config.attention_probs_dropout_prob,
                 hidden_dropout_ratio=config.hidden_dropout_prob,
                 num_hidden_layers=config.num_hidden_layers,
                 initializer_range=config.initializer_range,
+                local_rank=args.local_rank,
```

Comment:
is this required for 1bit adam? or accidental change?

Reply:
/cc @RezaYazdaniAminabadi

Reply:
the sequence-length is not anymore required, and yes the local-rank is needed in this config. The change sounds fine to me

Reply:
Thanks @RezaYazdaniAminabadi for reviewing. I will mark this as resolved then.